angular/src/lib/date-picker/date-picker-day.scss

```diff
@@ -0,0 +1,13 @@
+.selectedEdgeday {
```

Comment:
Sorry, maybe I'm not up to date on the angular lib, why would we have scss in angular?

Reply:
We do not have css in Core for the new component.

Reply:
@ArthusLiang  maybe add this to the Core?

Reply:
Will do this